### PR TITLE
Add compat data for HTMLAnchorElement

### PR DESCRIPTION
**Pull request by bunnybooboo**
https://developer.mozilla.org/en-US/docs/Web/API/HTMLAnchorElement

### CODE REVIEW

**Comment by bunnybooboo** [comment]
This file currently has 11 mixins omitted from the source page. Via discussion in issue #472 this requires further confirmation.

File: api/HTMLAnchorElement.json [hunk 1]
```diff
@@ -0,0 +1,916 @@
...
+            "safari": {
+              "version_added": true
+            },
+            "safari_ios": {
+              "version_added": true
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
```
Comment (Elchi3) [inline api/HTMLAnchorElement.json]: false

File: api/HTMLAnchorElement.json [hunk 2]
```diff
@@ -0,0 +1,916 @@
...
+            "deprecated": false
```
Comment (Elchi3) [inline api/HTMLAnchorElement.json]: true

File: api/HTMLAnchorElement.json [hunk 3]
```diff
@@ -0,0 +1,916 @@
...
+            "safari": {
+              "version_added": true
+            },
+            "safari_ios": {
+              "version_added": true
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
```
Comment (Elchi3) [inline api/HTMLAnchorElement.json]: false

File: api/HTMLAnchorElement.json [hunk 4]
```diff
@@ -0,0 +1,916 @@
...
+            "deprecated": false
```
Comment (Elchi3) [inline api/HTMLAnchorElement.json]: true

File: api/HTMLAnchorElement.json [hunk 5]
```diff
@@ -0,0 +1,916 @@
...
+            "safari": {
+              "version_added": true
+            },
+            "safari_ios": {
+              "version_added": true
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
```
Comment (Elchi3) [inline api/HTMLAnchorElement.json]: false

File: api/HTMLAnchorElement.json [hunk 6]
```diff
@@ -0,0 +1,916 @@
...
+            "deprecated": false
```
Comment (Elchi3) [inline api/HTMLAnchorElement.json]: true

File: api/HTMLAnchorElement.json [hunk 7]
```diff
@@ -0,0 +1,916 @@
...
+            "safari": {
+              "version_added": true
+            },
+            "safari_ios": {
+              "version_added": true
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
```
Comment (Elchi3) [inline api/HTMLAnchorElement.json]: false

File: api/HTMLAnchorElement.json [hunk 8]
```diff
@@ -0,0 +1,916 @@
...
+            "deprecated": false
```
Comment (Elchi3) [inline api/HTMLAnchorElement.json]: true

File: api/HTMLAnchorElement.json [hunk 9]
```diff
@@ -0,0 +1,916 @@
...
+            "safari": {
+              "version_added": true
+            },
+            "safari_ios": {
+              "version_added": true
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
```
Comment (Elchi3) [inline api/HTMLAnchorElement.json]: false

File: api/HTMLAnchorElement.json [hunk 10]
```diff
@@ -0,0 +1,916 @@
...
+            "deprecated": false
```
Comment (Elchi3) [inline api/HTMLAnchorElement.json]: true

File: api/HTMLAnchorElement.json [hunk 11]
```diff
@@ -0,0 +1,916 @@
...
+          }
+        }
+      },
+      "blur": {
```
Comment (Elchi3) [inline api/HTMLAnchorElement.json]: blur doesnt belong here. It belongs to HTMLElement. Please remove the entire entry.

File: api/HTMLAnchorElement.json [hunk 12]
```diff
@@ -0,0 +1,916 @@
...
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "focus": {
```
Comment (Elchi3) [inline api/HTMLAnchorElement.json]: focus doesnt belong here. It belongs to HTMLElement. Please remove the entire entry.

**Pull request merged**